### DISCUSSION TITLE
Which class is replacing MrSEQLClassifier()?

### DISCUSSION THREAD

**Discussion by brendan-gridsight** [alan-turing-institute/sktime#2032] [Q&A] [answered]
I'm currently receiving warnings related to the use of the `MrSEQLClassifier()` with sktime v0.9.0. I see that it has now been depreciated in v0.10.0 and that "`It will be replaced with an implementation based on Numba`".

What implementation is this? Is it currently available through `sktime`? Is there another suitable shapelet model for logistic classification?

Thanks.

**Comment by fkiraly** [comment]
@TonyBagnall, @chrisholder, @MatthewMiddlehurst?

**Comment by TonyBagnall** [comment, marked as answer]
hi, there is no specific plan to replace MrSEQL, although Georgiana (one of the authors) did express a desire to convert it. You could use the shapelet transform if you are after a pipeline of transform+logistic regressor. If you have a classification problem, I recommend using HIVECOTEv2 or ROCKET

**Reply by fkiraly** [reply]
shall we open an issue on "C-less MrSEQL"? It's a bit specific, but let's keep track of the todos.

**Reply by fkiraly** [reply]
I opened an issue https://github.com/alan-turing-institute/sktime/issues/2171
Implementation effortss are appreciated.

I guess @TonyBagnall's statement counts as an answer, so I'll mark it as such.
